Makefile

```diff
@@ -263,6 +264,12 @@ test-docker-e2e:
 	cd test/docker-e2e && go test -v -run ^TestCelestiaTestSuite/$(test)$$ ./...
 .PHONY: test-docker-e2e
 
+## test-docker-e2e-upgrade: Run the docker e2e upgrade test
+test-docker-e2e-upgrade:
+	@echo "--> Running docker e2e upgrade test"
+	cd test/docker-e2e && go test -v -run ^TestCelestiaTestSuite/TestCelestiaAppUpgrade$$ ./...
```

Comment:
we need to build it with the celestia tag before running the test by adding this line before this cmd:

`docker build -t celestiaorg/celestia-app:$(CELESTIA_TAG) -f docker/multiplexer.Dockerfile .`

Reply:
do we always want to build the image before a test run? If we are iterating on a test, the images will stay the same but the test will differ. At the moment it's possible to configure it via env vars, always building the image might slow it down a decent bit when the image does not need to be built. docker caching will help but  it might add time still